testkit/src/main/scala/scorex/testkit/properties/NodeViewSynchronizerTests.scala

```diff
@@ -59,9 +58,10 @@ trait NodeViewSynchronizerTests[P <: Proposition,
     }
   }
 
-  import NodeViewHolder._   // NodeViewHolder's messages
-  import NodeViewSynchronizer._   // NodeViewSynchronizer's messages
-  import NetworkController._      // NetworkController's messages
+  import scorex.core.LocalInterface.ReceivableMessages.{BetterNeighbourAppeared, NoBetterNeighbour}
+  import NodeViewHolder.ReceivableMessages.{ModifiersFromRemote, CompareViews}  // NodeViewHolder's messages
```

Comment:
we can now remove the comments from these lines, as they have become obvious.

Reply:
yep, indeed